Makefile

```diff
@@ -24,3 +24,28 @@ start_all:
 
 start_all_secure:
 	cd tools && ./start_all_tmux.sh secure
+
+start_test_all:
+	cd tools && chmod +x start_test.sh && ./start_test.sh
+
+clean_all_service:
```

Comment:
Well, I just found out that `cd tools && make clean` can clear up the built tools